scripts/nayduck.py

```diff
@@ -204,12 +209,16 @@ def run_locally(tests):
                 'expensive_tests',
                 '--',
                 '--exact',
-                fields[3])
+                fields[3]]
+            if timeout:
+                cmd = ["timeout", int(timeout)] + cmd
```

Comment:
Since this is the same in both branches, this should go outside before the `subprocess.check_call` invocation. You also don’t want to `int(timeout)` since `timeout` can have a suffix (like `1h`).